docs/howto/add_top_level_links.md

```diff
@@ -0,0 +1,40 @@
+# How to add top-level links
+
+JsonApi supports a [links object](http://jsonapi.org/format/#document-links) to be specified at top-level, that you can specify in the `render`:
+
+```ruby
+  links_object = {
+    href: "http://example.com/api/posts",
+    meta: {
+      count: 10
+    }
+  }
+  render json: @posts, links: links_object
+```
+
+That's the result:
+
+```json
+{
+  "data": [
+    {
+      "type": "posts",
+      "id": "1",
+      "attributes": {
+        "title": "JSON API is awesome!",
+        "body": "You should be using JSON API",
+        "created": "2015-05-22T14:56:29.000Z",
+        "updated": "2015-05-22T14:56:28.000Z"
+      }
+    }
+  ],
+  "links": {
+    "href": "http://example.com/api/posts",
+    "meta": {
+      "count": 10
+    }
+  }
+}
+```
+
+This feature is specific to JsonApi, so you have to use the use the [JsonApi Adapter](https://github.com/rails-api/active_model_serializers/blob/master/docs/general/adapters.md#jsonapi)
```

Comment:
link should be relative, `../general/adapters.md#jsonapi`. Also, I'm thinking this might be better placed in the Rendering, adapter_opts, `links` section (which didn't previously exist :( ) https://github.com/rails-api/active_model_serializers/blob/72c2c9f0d7c665ce1b32dbbd4ca5687b1afd320e/docs/general/rendering.md#links

thoughts on doing it now or in a follow-up PR?